native-messaging/app/ping_pong.py

```diff
@@ -18,7 +18,7 @@ def getMessage():
     # Encode a message for transmission,
     # given its content.
     def encodeMessage(messageContent):
-        encodedContent = json.dumps(messageContent).encode('utf-8')
+        encodedContent = json.dumps(messageContent, separators=(',', ':')).encode('utf-8')
```

Comment:
Please prepend a brief comment to explain that the separators argument exists to have a compact message, to reduce the odds of hitting the 1MB message size limit. Feel free to link to the github issue for context.